packages/ui-keyring/src/index.ts

```diff
@@ -32,7 +33,11 @@ const state: State = {
 
 loadAll(state);
 
+// FIXME The quicker we get in https://github.com/polkadot-js/apps/issues/138
```

Comment:
This can actually happen in parallel, quite happy to deal with the merge issues on both this and password to clean up this clutter. And it is not _that_ bad https://github.com/polkadot-js/common/pull/215

Reply:
I've added a couple of extra commits in the same PR #336 to address the classify ui-keyring Issue #138.